tools/settings.py

```diff
@@ -75,6 +75,16 @@ class CustomLogLevels:
     os.getenv("CURRENCY_CONVERTER_EXTRAPOLATION_MAX_DAYS_FALLBACK", 180)
 )
 
+FIELD_COVERAGE_STEP = "field_coverage"
+FIELD_QUALITY_STEP = "field_quality"
+DATASET_STEP = "dataset"
+TIME_BASED_STEP = "time_based"
+REPORT_STEP = "report"
+
+# A comma-separated list of steps to process. Default to all.
+PELICAN_STEPS = os.getenv(
```

Comment:
The internal setting can be `STEPS`, since everything within the project is "Pelican". The envvar can be `PELICAN_BACKEND_STEPS`.

Also, shouldn't we split on comma `,`? Right now we're using the `in` operator on a string, which has a different effect.